agent/agent.go

```diff
@@ -264,7 +263,8 @@ func initializeStateManager(cfg *config.Config, taskEngine engine.TaskEngine, cl
 		statemanager.AddSaveable("ContainerInstanceArn", containerInstanceArn),
 		statemanager.AddSaveable("Cluster", cluster),
 		statemanager.AddSaveable("EC2InstanceID", savedInstanceID),
-		statemanager.AddSaveable("ACSSeqNum", sequenceNumber),
+		//not tracking ACSSeqNum for now
```

Comment:
Updated the comment, will remove the field and bump the version after in future release (after I understand the implications of it). Thanks